charts/document-engine/values.yaml

```diff
@@ -76,7 +76,7 @@ apiAuth:
     # @section -- 1. API authentication
     secretKeyBaseKey: SECRET_KEY_BASE
 
-# -- (object) General configuration, see more in [our guide](https://pspdfkit.com/guides/document-engine/configuration/overview/)
+# -- (object) General configuration, see more in [our guide](https://www.nutrient.io/guides/document-engine/configuration/options/)
```

Comment:
All right, the `overview` options is very likely to be legit 😄

Reply:
It is.